src/main/java/org/sonarlint/intellij/config/project/SonarLintProjectBindPanel.java

```diff
@@ -275,7 +289,8 @@ public void actionPerformed(ActionEvent e) {
       WEST, HORIZONTAL, insets, 0, 0));
     bindPanel.add(configureConnectionButton, new GridBagConstraints(2, 3, 1, 1, 0.0, 0.0,
       WEST, HORIZONTAL, insets, 0, 0));
-
+    bindPanel.add(exportConfigurationButton, new GridBagConstraints(2, 2, 1, 1, 0.0, 0.0,
```

Comment:
I think we could add a link to the doc somewhere. It could be "?" next to the button or in the dialog message?